overlays/emacs.nix

```diff
@@ -82,9 +82,10 @@ let
             lib = drv: ''lib${libName drv}.${libSuffix}'';
             linkCmd = drv:
               if super.stdenv.isDarwin
-              then ''cp ${drv}/parser $out/lib/${lib drv}
-                     # FIXME: Is this kosher?
-                     /usr/bin/install_name_tool -id $out/lib/${lib drv} $out/lib/${lib drv}
+              then ''cp ${drv}/parser .
+                     chmod +w ./parser
+                     install_name_tool -id $out/lib/${lib drv} ./parser
+                     cp ./parser $out/lib/${lib drv}
                      /usr/bin/codesign -s - -f $out/lib/${lib drv}
```

Comment:
This is another dubious `/usr/bin` reference, but unlike `install_name_tool`, doesn't need Xcode.